modules/cluster-resources.adoc

```diff
@@ -0,0 +1,39 @@
+[id="support-cluster-resources_{context}"]
+= Interacting with your cluster resources
+
+You can interact with cluster resources by using the OpenShift CLI (`oc`) tool in {product-title}. The cluster resources that you see after running the `oc api-resources` command can be edited.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to the web console or you have installed the `oc` CLI tool.
+
+.Procedure
+
+. To see which configuration Operators have been applied, run the following command:
++
+[source,terminal]
+----
+$ oc api-resources -o name | grep config.openshift.io
+----
+
+. To see what cluster resources you can configure, run the following command:
++
+[source,terminal]
+----
+$ oc explain ${resource_name}.config.openshift.io
```

Comment:
Hi @xenolinux! Just a quick note that if you're noting something that the user replaces, we use the format `<resource_name>`. (As opposed to ${resource_name}). Not urgent, but if you wouldn't mind following up with a separate PR to adjust them, that would be appreciated!

Reply:
Thanks for noticing it @bergerhoffer. I opened a PR https://github.com/openshift/openshift-docs/pull/40670. Can you please take a look and merge if looks good?

Reply:
I believe QE ack won't be required for this minor change.

Reply:
Agreed on no QE needed, thanks @xenolinux, looking!